src-electron/main-process/startup.js

```diff
@@ -177,7 +177,12 @@ function gatherFiles(filesArg, options = { suffix: '.zap', doBlank: true }) {
  * @param {*} output
  */
 async function startConvert(argv, options) {
-  let files = argv.zapFiles
+  let zapFiles = argv.zapFiles
+  let files = gatherFiles(zapFiles, { suffix: '.zap', doBlank: true })
+  if (files.length == 0) {
+    options.logger(`    👎 no zap files found in: ${zapFiles}`)
+    throw `👎 no zap files found in: ${zapFiles}`
+  }
   let output = argv.output
```

Comment:
The argv.output was checked in the caller. Maybe it should be checked here now?

Reply:
Yes, I will add it to startConvert(). I am not sure what you mean about multiple files. The only change I made was deleting the check which simply makes sure the user did not forget to enter a required argument. Any other logic is unchanged. I guess we could add another check to make sure they did not add multiple files as an argument?